eq-author/src/App/page/Design/answers/MultipleChoiceAnswer/Option.js

```diff
@@ -65,6 +69,11 @@ export class StatelessOption extends Component {
     autoFocus: PropTypes.bool,
     label: PropTypes.string,
     getValidationError: PropTypes.func,
+    canMoveUp: PropTypes.bool,
+    onMoveUp: PropTypes.func,
+    canMoveDown: PropTypes.bool,
+    onMoveDown: PropTypes.func,
+    hideMoveButtons: PropTypes.bool,
```

Comment:
Should some of these be required?

Reply:
I don't think so. The generic option component can be used without them and checks are in place for their existence. The HOC Reorder component injects the props but option should work if used outside of Reorder.